src/xbps/repositories/custom.md

```diff
@@ -22,7 +22,7 @@ For example, to define a remote repository:
 # echo 'repository=http://my.domain.com/repo' > /etc/xbps.d/my-remote-repo.conf
 ```
 
-> Note: Remote repositories need to be [signed](signing.md).
+> Note: Remote repositories need to be [signed](./signing.md).
 > [xbps-install(1)](https://man.voidlinux.org/xbps-install.1) refuses to install
 > packages from remote repositories if they are not signed.
```

Comment:
This is probably something we should look at for #297 too!

Reply:
I have the blockquote removed in #297, but that's it. was there something else you wanted done to it?

Reply:
Just wanted to be sure :p You will probably need to rebase one of these, then.